### DISCUSSION TITLE
CH376msc Power Down

### DISCUSSION THREAD

**Discussion by jetcat-bob** [djuseeq/Ch376msc#68] [Q&A]
I noticed that the power down code CMD_ENTER_SLEEP is commented out in the .h file and there isn't a function to initiate it. Was this done intentionally because of incompatibility issues within the library? 

So far this library has worked great using a flash drive! Just looking for a way to conserve power when the drive is inactive.

**Comment by djuseeq** [comment]
Hi @ jetcat-bob. No, simply when I was writing this lib, I didn't need this feature. But saving energy is actually a good idea for battery powered devices.
